docs/source/introduction-to-mapreader/why-should-you-use-mapreader.rst

```diff
@@ -10,7 +10,16 @@ This exact number will vary depending on:
 - the skills you (or your team) have,
 - the amount of time at your disposal.
 
-Deciding to use MapReader, which uses deep learning computer vision (CV) models to predict the class of content on patches across many sheets, means weighing the pros and cons of working with the data output that is inferred by the model.
-Inferred data can be evaluated against expert-annotated data to understand its general quality (are all instances of a feature of interest identified by the model? does the model apply the correct label to that feature?), but in the full dataset there *will necessarily be* some percentage of error.
+MapReader uses computer vision (CV) models to extract information (class labels and text) from map images.
```

Comment:
Remove "extract" replace with "classify"

Reply:
then remove "class" from within brackets

Reply:
What about text spotting? I was trying to avoid using classify so we had a overall description of both classification and text spotting pipelines.